### ISSUE TITLE
Why people with different clothes count as one person(one ID)?

### ISSUE THREAD

**Issue by zhishao** [wanggrun/SYSU-30k#9]
In Market1501, DukeMTMC, MSMT17 , one person(one ID) all have same cloth and appearance, but in this dataset one dir have different person and different clothes, i think theres no feature the model can learn.

**Comment by wanggrun** [comment]
Q1: In Market1501, DukeMTMC, MSMT17, one person (one ID) has the same cloth and appearance, but in this dataset, one dir has different persons and clothes.
A1: Thank you very much for your interest in our dataset. Your description about "in this dataset one dir has different persons and clothes" might be incorrect. You might refer to bag_level_label, which is organized as follows.

Let's take a part of bag_level_label.json as an example for illustration:
[
...
{"BagLabel": [3], "BagID": "bag0000000009", "BagImages": ["0000000003/"]},
{"BagLabel": [2, 3, 4], "BagID": "bag0000000010", "BagImages": ["0000000002/", "0000000003/", "0000000004/"]},
{"BagLabel": [1, 2, 3, 4, 5], "BagID": "bag0000000011", "BagImages": ["0000000001/", "0000000002/", "0000000003/", "0000000004/", "0000000005/*"]},
...
]

This indicates that all the images in ["0000000003/"] can be assigned to a bag label [3]; and all the images in ["0000000002/", "0000000003/", "0000000004/"] can be assigned to a bag label [2, 3, 4]; and all the images in ["0000000001/", "0000000002/", "0000000003/", "0000000004/", "0000000005/*"] can be assigned to a bag label [1, 2, 3, 4, 5].

As shown, the rule of the organization is straightforward to follow. 


Q2: In this dataset, one dir has different persons and different clothes; I think there's no feature the model can learn.
A2: Even assuming that one dir has different persons and different clothes in this dataset, we can learn a powerful feature. We have performed an experiment by considering each dir as an ID and achieve fantastic high rank-1 accuracy for the SYSU-30k test set. We will provide this example in the directory of DistributedTrainSYSU30k in the future.

**Comment by zhishao** [comment]
@wanggrun Thx for your answer, I have some misunderstandings about the dataset ,  so, how to get the ID of single picture ?

**Comment by zhishao** [comment]
@wanggrun will you release **training code** for SYSU30k dataset ？

**Comment by wanggrun** [comment]
Dear @zhishao , thank you so much for your interest in our work. The training set of the SYSU-30k dataset focuses on weakly supervised learning and self-supervised learning. Therefore, there is no precise ID for a single image on the training set.

Best wishes to you,
